azure_arc_servers_jumpstart/gcp/terraform/scripts/deploy_arcagent.sh.tmpl

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Download the installation package
+wget https://aka.ms/azcmagent -O ~/install_linux_azcmagent.sh
+
+# Install the hybrid agent
+sudo bash ~/install_linux_azcmagent.sh
+
+# Import the config variables set in vars.sh
+source /tmp/vars.sh
+
+# Run connect command
+azcmagent connect \
+  --service-principal-id $TF_VAR_client_id \
+  --service-principal-secret $TF_VAR_client_secret \
+  --resource-group "${resourceGroup}" \
+  --tenant-id $TF_VAR_tenant_id \
+  --location "${location}" \
+  --subscription-id $TF_VAR_subscription_id
```

Comment:
change script name to "install_arc_agent.sh", same as in the other deployments in the repo.

Reply:
renamed as requested